cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           name: https
           protocol: TCP
       - name: metrics-server-nanny
-        image: k8s.gcr.io/addon-resizer:1.8.7
+        image: staging-k8s.gcr.io/addon-resizer:1.8.9
```

Comment:
Why use staging container image? This should be promoted to regular k8s.gcr.io and only then used.

Reply:
It's why this is WIP PR :-)